npc/jobs/2-2/sage.txt

```diff
@@ -1025,6 +1039,7 @@ L_AskQuestions:
 		}
 		close;
 	}
+	/*
```

Comment:
Done. Originally this would be matter of other commit, but since that comment is added in this pr and Ridley and you already requested it, I remove it.